lib/puppet/type/wls_datasource/connectionreservetimeoutseconds.rb

```diff
@@ -0,0 +1,12 @@
+newproperty(:connectionreservetimeoutseconds) do
+  include EasyType
+
+  desc 'The number of seconds after which a call to reserve a connection from the connection pool will timeout.'
+
+  defaultto '10'
```

Comment:
That's not how puppet is supposed to work though.  I can use other types to just manage the bits I want.
``` puppet resource file /foo mode='0600'``` only changes the file permissions.  It doesn't reset the ownership back to root etc.

I have applications whose installers create managed servers and datasources etc.  I then use puppet to modify the bits I need to.  With what you're suggesting, for each new orawls release I have to find out what new attributes have been added to which types and update my puppet code just to prevent puppet from making changes I didn't ask for.

Can you at least make this new behaviour configurable somehow?